src/CSSList/CSSList.php

```diff
@@ -296,6 +296,22 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Insert an item before its sibling.
+     *
+     * @param RuleSet|CSSList|Import|Charset $oItem The item.
+     * @param RuleSet|CSSList|Import|Charset $oSibling The sibling.
+     */
+    public function insertBefore($oItem, $oSibling): void
+    {
+        $iIndex = in_array($oSibling, $this->aContents);
```

Comment:
Also, this is a boolean (so the `i` prefix is misleading). I suggest to change the name altogether to allow it to communicate the intent: `$siblingExistsInList`

What do you think?

Reply:
Fixed.